### DISCUSSION TITLE
Installing and Uninstalling SLEAP

### DISCUSSION THREAD

**Discussion by wyvernEvelyn19** [talmolab/sleap#1996] [Help!] [answered]
I have managed to install SLEAP on the admin account but it has not been installed globally on other accounts. Do you have an idea why this might be happening? I also was looking at methods to uninstall SLEAP and could not find any on the website. Could you help me with this? I want to uninstall SLEAP and then try to install it globally on all the accounts on the PC.

**Comment by roomrys** [comment, marked as answer]
Hi @wyvernEvelyn19,

Is the mamba or [conda installation installed for all users](https://docs.anaconda.com/anaconda/install/multi-user/) (this is an option through the anaconda installer). If mamba or conda is installed for all users (and there is only a single mamba or conda installation), then each user should be able to access the `sleap` conda environment.

To [uninstall `sleap`](https://sleap.ai/installation.html#upgrading-and-uninstalling), you simply need to remove the conda or mamba environment that `sleap` is installed in:
```bash
conda env remove -n sleap
```
.

Let us know how it goes and/or if you need more guidance.

Thanks,
Liezl

P.S. Here is also a link for [uninstalling anaconda](https://docs.anaconda.com/anaconda/install/uninstall/) in case you have multiple anaconda installations (one per user) instead of a more global "all users" installation.
